tests/functional/workloads/ocp/logging/test_openshift-logging.py

```diff
@@ -81,86 +75,65 @@ def finalizer():
         )
         return pod_obj, pvc_obj
 
-    @retry(ModuleNotFoundError, tries=5, delay=200, backoff=1)
-    def validate_project_exists(self, project):
+    def setup_prerequisites(self, project):
         """
-        This function checks whether the new project exists in the
-        EFK stack
+        assign necessary permissions (full access) to service account
+        and generate token
 
         Args:
-            project (str): The project
+            project: test namespace to verify logging
 
+        Returns:
+               lokistack_route
+               decoded token
         """
```

Comment:
you should specify the return type with a description 

eg: 
 Returns:
        bool: The return value. True for success, False otherwise.